### DISCUSSION TITLE
How to configure CNAME for multiple github pages subdomains?

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#23513] [Other Feature Feedback, Questions, & Ideas]
<p>I would like to add a <code>status</code> subdomain to a repository on one of my organisations, but unfortunately do I already have a <code>docs</code> subdomain set up that points to the same <a href="http://github.io" rel="noopener nofollow ugc">github.io</a> subdomain.</p>
<p>So because of this was I wondering if I can set multiple subdomains to point to the <a href="http://github.io" rel="noopener nofollow ugc">github.io</a> subdomain using CNAME.<br>
Can I simply use a wildcard (*) and if so, what could be potential drawbacks/issues with it?</p>

**Comment by Bigson83** [comment]
<p>Hi there I am confused about what you are talking about</p>

**Comment by Andre601** [comment]
<p>I have a custom <code>docs</code> subdomain set up using CNAME and now would like to also add a <code>status</code> subdomain to another repository on the same organisation, but I can’t because CNAME can only have the same target once.<br>
So, is there a way I could allow multiple custom subdomains using CNAME (I.e. using a wildcard) and what would be the required steps for it (Including any security stuff needed to avoid abuse)?</p>
<p>Also, to be clear: I’m currently not able to add A-Records for GitHub Pages which from what I know could allow multiple subdomains, due to the domain being used on other places too.</p>

**Comment by Andre601** [comment, marked as answer]
<p>This is embarassing, but I have to mention it anyways…</p>
<p>I managed to solve the issue. Apparently did I still have an old CNAME entry for <code>status</code> that I didn’t delete, so the DNS manager (Cloudflare) yelled at me that it already was in use.<br>
Deleting the entry and adding the new one fixed the issue…</p>
